vinzenz/leapp-repository
# Coprcli makefile fix

Fixes in get_latest_copr_build:
    - fix shebang in helper script; use directly Python3 as the required
      module is supposed to exists only for Python3
    - accept PKG_RELEASE to ensure that we obtain the one required build;
      previously we would get just the latest build, but it could happen
      in real corner-case that we would obtain different build than we
      require and it would be hard to realize such thing
    - set the pagination to 10 (to retrieve 10 builds) to be able to
      find the required build for sure
    - set username as default owner when owner is not specified
      explicitly (through COPR_OWNER or as part of COPR_REPO);
      by default it is specified just the projectname through COPR_REPO,
      so in such case take the default value from the _COPR_CONFIG file
      as expected
    
    Fixes in Makefile:
    - provide PKG_RELEASE and _COPR_CONFIG to get_latest_copr_build;
      these values are not exported by default
    - store the build id into the file as $RELEASE contains already
      subshell and it is not possible to have nested subshells in
      Makefile
    - remove the obsoleted _list_approved_builds target
    
    Previous solution didn't work for default values, e.g.:
            make source
